test/ctia/task/check_es_stores_test.clj

```diff
@@ -102,6 +103,7 @@
                     "feedback - finished checking 0 documents"
                     "casebook - finished checking 100 documents"
                     "sighting - finished checking 100 documents"
+                    "identity-assertion - finished checking 0 documents"
```

Comment:
here you need to have the store check test validating the documents of your store, you might have missed the fixture import part for your entity.

Reply:
will be done in a separate PR